m3/thrift/v2/v2.thrift

```diff
@@ -0,0 +1,34 @@
+enum MetricType {
+    INVALID = 0
+    COUNTER = 1
+    GAUGE = 2
+    TIMER = 3
+}
+
+struct MetricValue {
+    1: required MetricType metricType
+    2: required i64 count
+    3: required double gauge
+    4: required i64 timer
```

Comment:
wait, we're sending each timer sample as a Metric payload, even back in V1 protocol?! can we at least make this a list in v2 thrift

Reply:
Discussed offline; will do this as a followup.